### PR TITLE
Add test to run an example DAG

### PR DESCRIPTION
**Pull request by kaxil**
Currently we don't a test where we run a DAG and a task. This will give us better coverage and prevent chart errors.

### CODE REVIEW

**Review by ernest-kr — APPROVED** [review]

File: bin/install-ci-tools
Comment (danielhoherd) [inline bin/install-ci-tools:9]: FWIW helm 3.6.0 just came out, and [it has a useful feature that we may be able to use in our purest setup](https://github.com/rikatz/kubepug/issues/139#issuecomment-850050037), so I'd recommend making an attempt to adopt it.

Comment (kaxil) [inline bin/install-ci-tools:9]:
```suggestion
  export HELM_VERSION="3.6.0"
```

Comment (kaxil) [reply bin/install-ci-tools]: Good call, giving it a shot -- just pushed the comment

File: tests/functional-tests/test_chart.py
Comment (danielhoherd) [inline tests/functional-tests/test_chart.py:10]: Please run `black` on this file or `pre-commit run --all-files`

Comment (kaxil) [reply tests/functional-tests/test_chart.py:10]: Pre-commit check passed, I have pre-commit installed and applied and CI passed too, no?

Comment (kaxil) [reply tests/functional-tests/test_chart.py:10]: Saw your other PR :) , thanks. Can we merge this before that so we can run pre-commit and do a mass refactor

Comment (kaxil) [reply tests/functional-tests/test_chart.py:10]: This PR still needs an approval from someone with merge permissions.

Comment (danielhoherd) [reply tests/functional-tests/test_chart.py:10]: FYI [I opened a PR to add pre-commit to CI](https://github.com/astronomer/airflow-chart/pull/223). We can probably hold off on fixing pre-commit errors in this branch, then fix any follow-ups in that PR before merging, just to avoid having merge conflicts.

File: tests/functional-tests/test_chart.py
Comment (danielhoherd) [inline tests/functional-tests/test_chart.py:207]: Would this be better? Then we could just get rid of `sleep_time`
```suggestion
        if try_count * sleep_time_between_polls >= timeout:
```

File: tests/functional-tests/test_chart.py
Comment (danielhoherd) [reply tests/functional-tests/test_chart.py:10]: Sorry, lagging here. You can merge it if you want. I was just trying to keep things easier for everybody but I don't have a preference on the order of merges.

**Review by danielhoherd — APPROVED** [review]
LGTM as is. There's one trivial optional change noted in the comments.

File: tests/functional-tests/test_chart.py
Comment (kaxil) [inline tests/functional-tests/test_chart.py:188]:
```suggestion
```

File: tests/functional-tests/test_chart.py
Comment (kaxil) [inline tests/functional-tests/test_chart.py:192]:
```suggestion
```

File: tests/functional-tests/test_chart.py
Comment (kaxil) [inline tests/functional-tests/test_chart.py:207]:
```suggestion
        if (try_count * sleep_time_between_polls) >= timeout:
```

Comment (kaxil) [reply tests/functional-tests/test_chart.py]: done in https://github.com/astronomer/airflow-chart/pull/222/commits/c7d7664f8010b12cefb5fe2fca8a8fa931c8d574

**Pull request merged**
